editoast/src/views/layers/geo_json.rs

```diff
@@ -0,0 +1,38 @@
+use mvt::GeomType;
+
+use rocket::serde::json::Value as JsonValue;
+use serde::{Deserialize, Serialize};
+
+/// GeoJson representation
+#[derive(Clone, Serialize, Deserialize, Debug)]
+#[serde(tag = "type", deny_unknown_fields)]
+pub enum GeoJson {
+    Point {
+        coordinates: (f64, f64),
+        crs: JsonValue,
```

Comment:
If we never use the `crs` field we can simply remove it and get rid of the `deny_unknown_fields` tag.